src/exe/main.cc

```diff
@@ -64,8 +64,7 @@ void maybe_slowdown_startup() {
   if (const char *s = getenv(k_startup_wait_ms_env_variable); s != nullptr) {
     std::string_view const sv{s};
     int wait_ms = 0;
-    auto [ptr, ec] = std::from_chars(sv.begin(), sv.end(), wait_ms);
-    if (ec == std::errc() && ptr == sv.end() && wait_ms > 0) {
+    if (absl::SimpleAtoi(s, &wait_ms) && wait_ms > 0) {
```

Comment:
Just for my information, what is the benefit of `absl::SimpleAtoi` against `std::from_chars` ?

Reply:
Well, `std::from_chars` is more powerful (it can be used to parse a number when the end of the number is not known since it returned the found end as a pointer) but it makes checking for success more difficult in simple cases when we expect the input string to a single number. And moreover it does not accept a `string`/ `string_view` as input.

Reply:
Got it, thx ! For instance with `"123 4"` `std::from_chars` will return `123` without error.

Reply:
`std::from_chars` would also return success for `123qwerty`.
 The other difference is that `absl::SimpleAtoi` will ignore leading whitespaces whereas those would cause `std::from_chars` to fail.